valgrind/docker/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:24.04
+
+RUN apt-get update && apt-get install -y git
```

Comment:
`apt update` pulls in a lot of data that sticks around on this layer, making the image uselessly bigger. It is common to `apt clean` after you have installed something eg: https://github.com/citusdata/citus/blob/117bd1d04fe55b4001012b3923216d5df0fda10e/.devcontainer/Dockerfile#L43C5-L43C14

There are a couple of more occurances of `apt update && apt install ...` down below

Reply:
I was mostly avoiding any extra steps to have a fast build, rather than optimizing for image size. But let me still do `apt clean` as it should not take a very long time.